product_docs/docs/pgd/5/proxy/index.mdx

```diff
@@ -0,0 +1,133 @@
+
+Proxy connects to BDR database for its internal operations, like getting proxy configs, getting write leader details, etc. Therefore, it needs a list of endpoints/dsn to connect to BDR nodes. Proxy expects these configurations in a local config file `pgd-proxy-config.yml`. Following is a functional example of `pgd-proxy-config.yml` file:
+
+```yaml
+log-level: debug
+cluster:
+  name: my_pgd_cluster
+  endpoints:
+    - "host=bdr-a1 port=5432 dbname=bdrdb user=pgdproxy "
+    - "host=bdr-a3 port=5432 dbname=bdrdb user=pgdproxy "
+    - "host=bdr-a2 port=5432 dbname=bdrdb user=pgdproxy "
+  proxy:
+    name: "proxy-a1"
+```
+
+The `cluster.endpoints` and `cluster.proxy.name` are mandatory fields in the config file. Proxy always tries to connect to the first endpoint in the list, if it fails it tries the next endpoint and so on. The log level for PGD Proxy service can be set using the top level config parameter `log-level` as shown in the sample config.
+
+PGD Proxy searches for `pgd-proxy-config.yml` in the following locations (precedence order - higher to lower):
+
+   1. "-f config-file" (e.g. 'pgd-proxy -f /opt/my-config.yml')
+   2. "/etc/edb/pgd-proxy" (default)
+   3. "$HOME/.edb/pgd-proxy"
+
+The `pgd-proxy-config.yml`, is located in the `/etc/edb/pgd-proxy` directory, by default. If you rename the file or move it to another location, specify the new name and location using the `-f` flag in the `pgd-proxy.service` file.
+
+### Proxy user
+
+If the PGD Cluster is created by TPAexec, then TPAExec creates a postgres user `pgdproxy` for PGD Proxy and `route_dsn` at node level is set. If you wish to setup a different user then you need to create the user and set in the `endpoints` in config file manually.
+
+
+## PGD Proxy service file
+
+PGD Proxy is preferably run as a systemd service. The `pgd-proxy` service unit file is located at `/etc/systemd/system/pgd-proxy.service` by default. Following is the sample service file for EDB Postgres Extended Server and Postgres Community.
+
+*Note:* For EDB Postgres Advanced Server please change User and Group from `postgres` to `enterprisedb`.
+
+```
+[Unit]
+Description=PGD Proxy
+
+[Service]
+Type=simple
+User=postgres
+Group=postgres
+Restart=on-failure
+RestartSec=1s
+ExecStart=/usr/bin/pgd-proxy -f /etc/edb/pgd-proxy/pgd-proxy-config.yml
+StandardOutput=syslog
+StandardError=syslog
+SyslogIdentifier=pgd-proxy
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Use the below commands to manage `pgd-proxy` service
+```
+systemctl status pgd-proxy
+systemctl stop pgd-proxy
+systemctl restart pgd-proxy
+```
+
+## Managing PGD Proxy
+
+PGD CLI provides few commands viz., `create-proxy`, `show-proxy`, `set-proxy-options` and `delete-proxy` to manage Proxies in PGD cluster. See [PGD CLI](/product_docs/docs/pgd/5/cli/command_ref) for more information.
+
+See [Connection management](/product_docs/docs/pgd/5/routing) for more information on BDR side of configuration and management of PGD Proxy.
+
```

Comment:
What about monitoring?

Reply:
We don't have any monitoring capabilities as yet for proxy. if you are talking about the http probes then AFAIU it was added for CNP team and is an internal feature so I don't think  we would want to announce that in public docs. There are still open questions around that and we might remove that in future.

Reply:
I think the probe should be documented, but will merge this into dev/pgd5 anyway, we can do another PR later